lib/watir/elements/input.rb

```diff
@@ -0,0 +1,14 @@
+module Watir
+  class Input < HTMLElement
+
+    #
+    # Returns applicable label.
+    # Is not lazy loaded.
+    #
+
+    def label
+      parent(tag_name: 'form').label(for: id)
```

Comment:
Hrmm... so ideally we would want the ancestor `form` if available, otherwise the `body`. Probably not worth addressing until someone runs into the issue.